networking/multiple_networks/understanding-user-defined-network.adoc

```diff
@@ -0,0 +1,65 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="understanding-user-defined-networks"]
+= Understanding user-defined networks
+include::_attributes/common-attributes.adoc[]
+:context: understanding-user-defined-networks
+
+toc::[]
+
+:featurename: {`UserDefinedNetwork`}
+include::snippets/technology-preview.adoc[]
+
+The OVN-Kubernetes CNI plugin has supported `layer3`, `layer2`, and `localnet` topology types for creating additional networks in which secondary interfaces of pods are attached, and for a primary network a pod only `layer3` topology type was supported.
+This allowed for network models where all pods in the cluster were part of the same global `layer3` network, but restricted the ability to customize primary network configurations.
+
+With user-defined networks, cluster administrators and users are offered highly customizable network configuration options that provide users with the ability to define their own network topologies, ensure network isolation, manage IP addressing for the workloads, and leverage advanced networking features. User-defined networks help enhance networking flexibility, security, and performance.
+
+User-defined networks provide the following benefits:
```

Comment:
@tssurya should we mention the self-service capability we have by the namespace UDN CRD?

Reply:
wdym by self-service? ah tenants can create networks?  that should go into the API section...when we start talking about the UDN namespace API? then in 4.18 when CUDN comes we can add on top the admin persona

Reply:
The APIs are automatically generated, so changes to them should go through the api repo.